bindgen-integration/Cargo.toml

```diff
@@ -11,4 +11,4 @@ bindgen = { path = ".." }
 gcc = "0.3"
 
 [features]
-llvm_stable = ["bindgen/llvm_stable"]
+__testing_only_llvm_stable = ["bindgen/__testing_only_llvm_stable"]
```

Comment:
IIRC, `cargo` had issues with `_` prefixed features, so we may need to change this to simply `testing_only_llvm_stable`. We'll see when travis CI results get in :)